docs/basics/connection-approval.md

```diff
@@ -61,13 +62,64 @@ private void ApprovalCheck(NetworkManager.ConnectionApprovalRequest request, Net
 
     // Rotation to spawn the player object (if null it uses the default of Quaternion.identity)
     response.Rotation = Quaternion.identity;
+    
+    // If response.Approved is false, you can provide a message that explains the reason why via ConnectionApprovalResponse.Reason
+    // On the client-side, NetworkManager.DisconnectReason will be populated with this message via DisconnectReasonMessage
+    response.Reason = "Some reason for not approving the client";
```

Comment:
I'm not sure about the default value. Is this a case where we get support tickets "Strange reason pops up even though the server did not provide it" ?